src/app/auth/shared/store/errors/errors.reducer.ts

```diff
@@ -0,0 +1,15 @@
+import { on, createReducer } from '@ngrx/store';
+import { errorsAction } from './errors.actions';
+
+export const initialState = '';
+
+const errorsSubReducer = createReducer(
+  initialState,
+  on(errorsAction, (state, action) => {
+    return action.error;
+  })
+);
+
+export function errorsReducer(state, action) {
+  return errorsSubReducer(state, action);
+}
```

Comment:
if you're going to continue to create reducer per one action, soon you'll have dozens reducers. Also, see comments above regarding this.